### ISSUE TITLE
Error message where there should be Value feedback message

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/webwork2#1232]
Carl Yao posted an issue in the forums:
https://webwork.maa.org/moodle/mod/forum/discuss.php?d=5035

To investigate, I took his code and created a .pg file in a course. To reproduce the issue I will describe below (which is separate from Carl's issue), I suggest starting by doing the same thing.

Now make a new problem set with one question, and set it to use this problem. Set the open date so that you may submit answers. Go to the problem and it should be rendering without error at this point. Now enter "(0,0)" and submit. You will get an error message, where you should have gotten a message about the submitted answer being a list of numbers (or a point, or something like that) instead of a formula. This is part 1 of the issue.

Part 2. Now make sure that you have permission "Can show old answers". Then return to the problem page with a fresh reload. Now I still get that error message, even though I haven't submitted anything this round. The previous answer is somehow triggering the PG error. If I take away permission "Can show old answers" and return to the problem, then there is no error.

So I see two issues here. To me, part 2 is more concerning. When I make a fresh visit to the problem, why is anything from the previous answer relevant? (Aside from just populating the input field with the text of the previous answer.) Evidently, "Can show old answers" is about more than just "showing" old answers. 

Part 1 is also concerning. The problem has the ad hoc creation of a complex-valued function, but I can't see anything "off" with how that is handled. 

Probably fixing part 1 would make part 2 moot. But again I wonder why a previous answer from a previous session should impact the rendering here.

**Comment by somiaj** [comment]
@Alex-Jordan If I use the 'broken problem' from that post I do get the error, "Can't generate enough valid points for comparison at line 583 of [PG]/lib/AnswerHash.pm", but if I 'fix' the problem using the suggestion to change `Formula` to `Compute`, it works as expected. So the error was due to an error in the problem that needed to fixed that I could see.

I'm unsure if part 2 is an issue. the `can show old answers` permission means that is there is a past answer, when viewing the problem (epically now that the answer table is gone) the past answer is checked. So any errors/feedback are available when viewing the problem, so I don't think part 2 should be considered an issue.

Anyways, I'm inclined to close this. There are things that could be done to improve pg errors/feedback, but that issue should be moved to pg. I think this old issue should be closed, but will leave it up to you to decide.

**Comment by Alex-Jordan** [comment]
I agree that it is a PG issue. Moving it to there would be ok, but I don't
want to close it yet. I don't feel that using Formula is an authoring error
here. Also, this is the kind of thing to sic Claude on. Other projects I'm
involved with have been using Claude to triage issue report backlogs with
success, and sooner or later if like to try that with webwork2 and pg.

Alex Jordan
Mathematics Instructor
Portland Community College
